tox.ini

```diff
@@ -12,6 +12,7 @@ deps=
   nose<2.0
   unittest2<2.0
   boto<3.0
```

Comment:
I know nothing about boto. But is it possible to remove `boto` dependency now that you introduce `boto3`?